lib/screens/profile_screens/security/security_screen.dart

```diff
@@ -81,6 +82,16 @@ class SecurityScreen extends StatelessWidget {
                           );
                         },
                       ),
+                      if (state.shouldShowExportRecoveryPhrase)
```

Comment:
remoteConfigurations - like any other service, shall not be accessed at the UI level. 
Instead we add a new value to the state to represent this.